Fomo/User.swift

```diff
@@ -10,6 +10,18 @@ import UIKit
 var _currentUser: User?
 
 class User: NSObject {
+    
+    enum Response: Int {
+        case Accepted = 0, Declined, Maybe, Invited
+    }
+    
+    var id: Int?
+    var name: String?
+    var email: String?
+    var avatarImageUrl: String?
+    var preferences: [AttractionType]?
+    var response: Response?
+    
```

Comment:
is the friends list as imported from Facebook stored on the User object? (not sure how importing contacts usually works)